libs/domain/site/src/services/jsonld/jsonld.normalizer.ts

```diff
@@ -0,0 +1,21 @@
+import { Observable } from 'rxjs';
+import { JSONLD } from './jsonld.model';
+
+export interface JsonLdNormalizer<T = any> {
```

Comment:
What about reusing existing transformer infra for it (what we use for adapter normalizers)?